packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -762,7 +758,7 @@ function updateCacheComponent(
     pushCacheProvider(workInProgress, freshCache);
   } else {
     // Check for updates
-    if (includesSomeLane(renderLanes, updateLanes)) {
+    if (includesSomeLane(current.lanes, renderLanes)) {
```

Comment:
The order for this function doesn't really matter because the implementation is `A & B !== 0`, and we're not super consistent elsewhere, but to explain my thinking, I usually try to put them in  "subject-object" order, like English. So  this would be: "check if the update lanes include the render lanes." Although you could argue either one is the "subject", but for whatever reason that's how I think of it. I guess because in a typical scenario the update lanes are a larger set than the render lanes.

Reply:
Makes sense!